packages/citation-js-utils/src/index.ts

```diff
@@ -1,11 +1,15 @@
-import { Cite } from '@citation-js/core';
+import { Cite, plugins } from '@citation-js/core';
 import { doi as doiUtils } from 'doi-utils';
 import { clean as cleanCSL } from '@citation-js/core/lib/plugins/input/csl.js';
 import sanitizeHtml from 'sanitize-html';
 
 import '@citation-js/plugin-bibtex';
 import '@citation-js/plugin-csl';
 
+const config = plugins.config.get('@bibtex');
+config.format.useIdAsLabel = true;
+config.format.checkLabel = false;
```

Comment:
If `checkLabel` is not explicitly set to `false`, punctuation is removed, causing the case from #1318 to still fail.